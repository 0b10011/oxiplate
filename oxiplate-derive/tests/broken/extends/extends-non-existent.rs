use oxiplate_derive::{Oxiplate};

#[derive(Oxiplate)]
#[oxiplate = "extends-missing.html.oxip"]
struct Data;

fn main() {
    assert_eq!("", format!("{}", Data));
}
