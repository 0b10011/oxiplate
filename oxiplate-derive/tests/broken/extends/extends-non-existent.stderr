error: proc-macro derive panicked
 --> tests/broken/extends/extends-non-existent.rs:3:10
  |
3 | #[derive(Oxiplate)]
  |          ^^^^^^^^
  |
  = help: message: Template path could not be canonicalized; Provided: extends-non-existent.html.oxip

error[E0277]: `Data` doesn't implement `std::fmt::Display`
 --> tests/broken/extends/extends-non-existent.rs:8:34
  |
8 |     assert_eq!("", format!("{}", Data));
  |                             --   ^^^^ `Data` cannot be formatted with the default formatter
  |                             |
  |                             required by this formatting parameter
  |
  = help: the trait `std::fmt::Display` is not implemented for `Data`
  = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
  = note: this error originates in the macro `$crate::__export::format_args` which comes from the expansion of the macro `format` (in Nightly builds, run with -Z macro-backtrace for more info)
